### DISCUSSION TITLE
Adonis validator: no url for rules?

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#1893] [Help]
With Indicative, we had a `url` rule we could use to make sure the sent parameter was a URL.

It seems like this is not working with Adonis V5. Is it an intended choice?

```javascript
export default class Validator {
  public schema = schema.create({
    website: schema.string({}, [rules.url()]),  // rules.url() does not exist
  })
}
```

**Comment by thetutlage** [comment]
Nope its not. Happy to accept a PR if you want to add this rule :)

**Reply by LeCoupa** [reply]
Sure, will do it this weekend!
